core/google/cloud/iam.py

```diff
@@ -154,47 +178,10 @@ def from_api_repr(cls, resource):
         policy = cls(etag, version)
         for binding in resource.get('bindings', ()):
             role = binding['role']
-            members = set(binding['members'])
-            if role in cls._OWNER_ROLES:
-                policy.owners |= members
-            elif role in cls._EDITOR_ROLES:
-                policy.editors |= members
-            elif role in cls._VIEWER_ROLES:
-                policy.viewers |= members
-            else:
-                policy._bind_custom_role(role, members)
+            members = sorted(binding['members'])
```

Comment:
Is there a reason you're not continuing to use `set` here? It will be an error to specify the same member twice on the API side, so a set is very helpful.